.github/workflows/test.yml

```
@@ -155,3 +155,135 @@ jobs:
       run: python manage.py test
       env:
         DATABASE_URL: sqlite://localhost/testdb.sqlite
+
+  django-main-sqlite:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.11']
+        requirements-file: ['requirements_base.txt']
+        django-version: [
+          'https://github.com/django/django/archive/main.tar.gz'
+        ]
+        os: [
+          ubuntu-20.04,
+        ]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python ${{ matrix.python-version }}
+
+      uses: actions/setup-python@v4
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        sudo apt install gettext gcc -y
+        python -m pip install --upgrade pip
+        pip install pytest ${{ matrix.django-version }}
+        pip install -r test_requirements/${{ matrix.requirements-file }}
+        python setup.py install
+
+    - name: Test with django test runner
+      run: python manage.py test
+      continue-on-error: true
+      env:
+        DATABASE_URL: sqlite://localhost/testdb.sqlite
+
+  django-main-postgres:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.11']
+        requirements-file: ['requirements_base.txt']
+        django-version: [
+          'https://github.com/django/django/archive/main.tar.gz'
+        ]
+        os: [
+          ubuntu-20.04,
+        ]
+
+    services:
+      postgres:
+        image: postgres:12
```

```suggestion
        image: postgres:13
```
Django main dropped postgres 12